Project.toml

```diff
@@ -23,6 +23,7 @@ PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
 REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 RegistryInstances = "2792f1a3-b283-48e8-9a74-f99dce5104f3"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
+TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
```

Comment:
If we don't want to add the standard library `TOML` dependency for some reason, I can also use the same regex-based extraction as in

https://github.com/JuliaDocs/Documenter.jl/blob/deecbd7d3abc395fe31d7a393539bcda4c00d753/src/Documenter.jl#L32-L38

Using TOML seems pretty unproblematic, though. In fact, it would probably be fine to use for the definition of `DOCUMENTER_VERSION`